server/block/grass.go

```diff
@@ -78,3 +92,30 @@ func (g Grass) Till() (world.Block, bool) {
 func (g Grass) Shovel() (world.Block, bool) {
 	return DirtPath{}, true
 }
+
+// plantSelection returns the selection of plants that can be grown by bone meal.
+func plantSelection() []world.Block {
```

Comment:
You could simply store this in a package level variable so you don't need to recreate this slice every time bone meal is used on grass.